internal/cmd/refactortests/cmd/refactortests_test.go

```diff
@@ -0,0 +1,230 @@
+	script := `
+import "testing"
+
+inData = "
+#datatype,string,long
+#group,false,false
+#default,_result,0
+,result,table
+"
+
+outData = "
+#datatype,string,long
+#group,false,false
+#default,_result,0
+,result,table
+"
+
+test t = () => ({
+    input: testing.loadMem(csv: inData),
+    want: testing.loadMem(csv: outData),
+    fn: (table=<-) => table
+})
+`
+
+	pkg := parser.ParseSource(script)
+	if ast.Check(pkg) > 0 {
+		if err := ast.GetError(pkg); err != nil {
+			t.Fatal(err)
+		}
+	}
+
+	oldPkg := pkg.Copy().(*ast.Package)
+
+	newPkg, err := inPlaceTestGen(pkg)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// assert old package hasn't changed
+	if !cmp.Equal(pkg, oldPkg, asttest.IgnoreBaseNodeOptions...) {
+		t.Errorf("original AST has changed: -want/+got:\n%s", cmp.Diff(pkg, oldPkg, asttest.IgnoreBaseNodeOptions...))
+	}
+
+	pattern := getRunTestPattern("t")
+	matched := edit.Match(newPkg, pattern, true)
+	if len(matched) != 1 {
+		t.Errorf("couldn't get exact match for running test statement, got %d, want: %d", len(matched), 1)
+	}
+}
+
+func Test_InPlaceGen_MultipleTestCases(t *testing.T) {
+	script := `
+import "testing"
+
+inData = "
+#datatype,string,long
+#group,false,false
+#default,_result,0
+,result,table
+"
+
+outData = "
+#datatype,string,long
+#group,false,false
+#default,_result,0
+,result,table
+"
+
+test a = () => ({
+    input: testing.loadMem(csv: inData),
+    want: testing.loadMem(csv: outData),
+    fn: (table=<-) => table
+})
+test b = () => ({
+    input: testing.loadMem(csv: inData),
+    want: testing.loadMem(csv: outData),
+    fn: (table=<-) => table
+})
+test c = () => ({
+    input: testing.loadMem(csv: inData),
+    want: testing.loadMem(csv: outData),
+    fn: (table=<-) => table
+})
+test d = () => ({
+    input: testing.loadMem(csv: inData),
+    want: testing.loadMem(csv: outData),
+    fn: (table=<-) => table
+})
+test e = () => ({
+    input: testing.loadMem(csv: inData),
+    want: testing.loadMem(csv: outData),
+    fn: (table=<-) => table
+})
+`
+
+	pkg := parser.ParseSource(script)
+	if ast.Check(pkg) > 0 {
+		if err := ast.GetError(pkg); err != nil {
+			t.Fatal(err)
+		}
+	}
+
+	newPkg, err := inPlaceTestGen(pkg)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	for _, tcName := range []string{"a", "b", "c", "d", "e"} {
+		pattern := getRunTestPattern(tcName)
+		matched := edit.Match(newPkg, pattern, true)
+		if len(matched) != 1 {
+			t.Errorf("couldn't get exact match for running test statement, got %d, want: %d", len(matched), 1)
+		}
+	}
+}
+
+func Test_GeneratesARunningTest_WithTestStatement(t *testing.T) {
+	script := `
+import "testing"
+
+inData = "
+#datatype,string,long
+#group,false,false
+#default,_result,0
+,result,table
+"
+
+outData = "
+#datatype,string,long
+#group,false,false
+#default,_result,0
+,result,table
+"
+
+test t = () => ({
+    input: testing.loadMem(csv: inData),
+    want: testing.loadMem(csv: outData),
+    fn: (table=<-) => table
+})
+
+testing.run(case: t)
```

Comment:
I think this line shouldn't be here. The objective was to see if a test without that line runs.